src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -13,7 +14,8 @@ public static string GetPrintableEnvironmentVariables(IEnumerable<KeyValuePair<s
 
             if (variables != null)
             {
-                foreach (var pair in variables)
+                var sortedVariables = variables.OrderBy(x => x.Key, StringComparer.OrdinalIgnoreCase);
```

Comment:
one suggestion is to have an optional sort parameter, if someone does not want to print in sort then it is fine. is it not?

also we are sorting the enumerable, instead, why can not we take only the keys and sort them and then take the value of from the key. 
foreach(var key in sortedKeys) {
dict[key]
}

something like above